test/unit_tests/modules/classes/BreakpointHandler-spec.js

```diff
@@ -10,8 +10,8 @@ var BASE_LOC = '../../../../cfgov/unprocessed/js/';
 
 beforeEach( function() {
   args = {
-    enter:      function() {},
-    leave:      function() {},
+    enter:      function() { return; },
```

Comment:
Added noop in https://github.com/cfpb/cfgov-refresh/commit/293473e2d3156a219db95bd473a4e629221887b4